quests/the_forge.md

```diff
@@ -251,10 +251,10 @@ class pasture (
 ```
 
 Next, edit the `pasture_config.yaml.epp` template. We'll use a conditional
-statement to only include the `:db_uri:` setting if there is a value set for
+statement to only include the `:uri:` setting if there is a value set for
```

Comment:
I made a similar change in the version of the file that I'm working on, but I created a class parameter named `db`, because it looks like the pasture config file has a `:db:` setting. 

I looked at the pasture source code, and it seems like it expects a configuration parameter named `db`, so I standardized on that name for the class parameter and template parameter. Does that sound right?